scripts/target/mame/mess.lua

```diff
@@ -3006,6 +3006,11 @@ files {
 	MAME_DIR .. "src/mame/includes/tmc2000e.h",
 }
 
+createMESSProjects(_target, _subtarget, "teletype")
+files {
+	MAME_DIR .. "src/mame/drivers/att4425.cpp",
+}
+
```

Comment:
Can you please put this in the skeleton project for now?  We already have a plethora of single-driver projects.